### PR TITLE
OD-356 [Fix] Users will be able to work with accessibility

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 

## Issue
https://weboo.atlassian.net/browse/OD-356

## Description
Fixed accessibility issues from the table.

## Screenshots/screencasts
https://share.getcloudapp.com/RBu4BlWG

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko @ivandevupp @AndrRyaz

### CODE REVIEW

File: css/menu.css [hunk 1]
```diff
@@ -307,3 +307,7 @@ body.disableScroll {
     height: auto;
   }
 }
+
+.hidden {
```
Comment (tonytlwu) [inline css/menu.css:311]: Why is this needed?

image

Bootstrap already has this exact same style.

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: Low
Affected: App users with the Bottom Bar menu
Verify: Users can access all menu items as expected

Squash and merge: Enhance the accessibility of menu

File: css/menu.css (hunk 1 above)
Comment (tonytlwu) [inline css/menu.css:311]: Why is this needed?

image

Bootstrap already has this exact same style.

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: css/menu.css (hunk 1 above)
Comment (YaroslavOvdii) [reply css/menu.css]: @tonytlwu removed unneeded CSS

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing on staging.

Risk: Low
Affected: App users with the Bottom Bar menu
Verify: Users can access all menu items as expected

Squash and merge: Enhance the accessibility of menu
